test/validation/api/event/event.c

```diff
@@ -309,7 +309,7 @@ static void type_test_init(odp_pool_t *buf_pool, odp_pool_t *pkt_pool,
 static void event_test_type_multi(void)
 {
 	odp_pool_t buf_pool, pkt_pool;
-	odp_event_type_t type;
+	odp_event_type_t type = 0;
```

Comment:
Safer to use a valid event type value (e.g. `ODP_EVENT_PACKET`).

Reply:
Fixed in V2